Tests/HarmonyTests/GenericNetworkDataSourceUtils.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  GenericNetworkDataSourceUtils.swift
+//  HarmonyTests
+//
+//  Created by Borja Arias Drake on 11.10.2022..
+//
+
+import Foundation
+import Alamofire
+
+@available(iOS 13.0, *)
+final class GenericDataSourceUtils {
+    static func provideRequest(url: String, cachePolicy: URLRequest.CachePolicy, timeout: TimeInterval) -> URLRequest {
+        return URLRequest(url: URL(fileURLWithPath: url), cachePolicy: cachePolicy, timeoutInterval: timeout)
+    }
+
+    static func provideResponse(url: String, statusCode: Int, httpVersion: String, headers: [String: String]) -> URLResponse? {
```

Comment:
It's our `anyURLResponse(..)` pattern for testing.

Reply:
done